actionlist/__tests__/index.android.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
No tests are present currently so we have not linted this. Stephen would be changing this in the testing branch changes